examples/react-app/package.json

```diff
@@ -42,6 +42,10 @@
     ]
   },
   "devDependencies": {
+    "@aws-sdk/client-iottwinmaker": "file:~/workplace/AppKitV5/aws-sdk-client-iottwinmaker-3.571.0.tgz",
```

Comment:
Why are you having to modify `react-app`?

Reply:
local library file issues.  Hopefully I can revert this in the final PR version